ingest/usecase/plugins/orderbook/claimbot/plugin.go

```diff
@@ -0,0 +1,186 @@
+package claimbot
+
+import (
+	"context"
+	"fmt"
+	"sync/atomic"
+
+	"github.com/osmosis-labs/sqs/domain"
+	"github.com/osmosis-labs/sqs/domain/keyring"
+	"github.com/osmosis-labs/sqs/domain/mvc"
+	orderbookdomain "github.com/osmosis-labs/sqs/domain/orderbook"
+	"github.com/osmosis-labs/sqs/domain/slices"
+	"github.com/osmosis-labs/sqs/log"
+
+	"github.com/osmosis-labs/osmosis/osmomath"
+
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+
+	"go.opentelemetry.io/otel"
+	"go.uber.org/zap"
+)
+
+// claimbot is a claim bot that processes and claims eligible orderbook orders at the end of each block.
+// Claimable orders are determined based on order filled percentage that is handled with fillThreshold package level variable.
+type claimbot struct {
+	config     *Config
+	atomicBool atomic.Bool
+}
+
+var _ domain.EndBlockProcessPlugin = &claimbot{}
+
+const (
+	tracerName = "sqs-orderbook-claimbot"
+)
+
+var (
+	tracer        = otel.Tracer(tracerName)
+	fillThreshold = osmomath.MustNewDecFromStr("0.98")
```

Comment:
Dito. Consider having a const representing 0.98

Reply:
Or even in config